frontend/micro-ui/web/micro-ui-internals/packages/components-core/src/atoms/CardSectionHeader.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const CardSectionHeader = (props) => {
+  return (
+    <header id={props.id} className="card-section-header" style={props.style}>
+      {props.children}
```

Comment:
make all card header, subheader as a generic , card header component and use different class names which or a variant way.